packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,181 @@
+        <span className="flex flex-col laptop:flex-row gap-3 laptopL:gap-6 items-center laptop:items-start mb-7 tablet:mb-8">
+          <ProfileImageLink
+            user={referringUser}
+            picture={{ size: isLaptopL ? 'xxxlarge' : 'xlarge' }}
+          />
+          <p className="my-auto text-center laptop:text-left typo-headline laptop:typo-title3 laptopL:typo-title2">
+            <span className="block laptop:inline">{referringUser.name}</span>
+            <span className="font-normal"> invited you to daily.dev</span>
+          </p>
+        </span>
+        <h1 className="w-full text-center laptop:text-left break-words-overflow typo-large-title tablet:typo-mega2 laptopL:typo-giga2">
+          The homepage
+          <span className="block font-bold text-theme-color-cabbage">
+            {' '}
+            developers deserve
+          </span>
+        </h1>
+        <p className="mx-auto laptop:mx-0 mt-7 tablet:mt-8 max-w-sm laptopL:max-w-xl text-center laptop:text-left typo-title3 text-theme-label-tertiary">
+          Get a feed of the best developer news out there! Read more quality
+          articles.
+          <span className="tablet:block laptopL:inline">Stay up to date.</span>
+        </p>
+        <Button
+          buttonSize={ButtonSize.Large}
+          // important has been used is some classnames to override the default styles as agreed on [thread](https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1699023366663489?thread_ts=1699011522.350609&cid=C05P9ET7S9K)
+          className="p-4 mx-auto laptop:mx-0 mt-6 tablet:mt-12 max-w-[17.5rem] mobileL:max-w-[fit-content] btn-primary mobileL:!h-[4rem] mobileL:!p-[0] mobileL:!px-[1.5rem]"
+          tag="a"
+          href={downloadBrowserExtension}
+          onClick={() => {
+            handleAcceptClick();
+          }}
+          target="_blank"
+          rel={anchorDefaultRel}
+          loading={isLoading}
+          disabled={isLoading || isSuccess}
+        >
+          <FlexCentered className="gap-2 mobileL:typo-title3">
+            <BrowsersIcon className="text-theme-label-primary w-[3.7rem] h-[1.9rem] mobileL:w-[5rem] mobileL:h-[2.5rem]" />
+            Try it now - It&apos;s free
+          </FlexCentered>
+        </Button>
+      </div>
+      <div className="flex fixed -bottom-1/2 flex-auto w-full h-full laptop:initial laptop:bottom-[unset]">
+        <div className="flex relative laptop:absolute laptopL:-right-24 justify-center self-center m-auto mx-2 z-10 laptop:right-[-7.5rem] w-fit laptop:w-[initial]">
+          <img
+            src={
+              cloudinary.referralCampaign.genericReferral.purpleEdgeGlowTablet
+            }
+            alt="Purple glow right edge"
+            className="fixed laptop:top-0 -bottom-4 w-full laptop:w-auto laptop:h-full laptop:bottom-[unset] laptop:rotate-[270deg] z-20"
```

Comment:
I think we can use the negative value.
```suggestion
            className="fixed laptop:top-0 -bottom-4 w-full laptop:w-auto laptop:h-full laptop:bottom-[unset] laptop:-rotate-90 z-20"
```

Reply:
True, thank u!!